hashicorp/boundary-ui
# chore: 🤖 upgrade braces to 3.0.3 to fix security vulnerability

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14205

# Description

* Removed `braces@^3.0.2, braces@~3.0.2` yarn.lock entry to force it to resolve to 3.0.3
* Removed `micromatch@^4.0.2, micromatch@^4.0.4, micromatch@^4.0.5` to force braces to ^3.0.3
* Upgraded `ember-intl` to newest version (v7)
    * `* braces@^2.3.1 -> micromatch@^3.1.10 -> fast-glob@^2.2.6 -> broccoli-merge-files@^0.8.0 -> ember-intl@^6.0.0-beta.6`
    * followed this [migration guide](https://ember-intl.github.io/ember-intl/docs/migration/v7)
    * note newest `ember-intl` v6 resolution also fixed this but we should upgrade to newest either way
* Last reference was `braces@^2.3.1` and had to add a global resolution for this one
    * `* braces@^2.3.1 -> micromatch@^3.1.4 -> find-yarn-workspace-root@^1.1.0 -> ember-cli-dependency-checker@^3.3.1`
* Fixes dependabot alerts: [#98](https://github.com/hashicorp/boundary-ui-enterprise/security/dependabot/98) && [#147](https://github.com/hashicorp/boundary-ui/security/dependabot/147)


## How to Test

1. `yarn start` in `ui/admin`
2. `yarn start:desktop` in `ui/desktop`
3. [boundary-ui-releases test run](https://github.com/hashicorp/boundary-ui-releases/actions/runs/10711594647)
4. run `yarn install` locally and make sure there are no NEW warnings.

## Checklist
